index.html

```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
         <li>
           <a href="#"><img src="images/Vector.svg" alt="thumb" /></a>
         </li>
+        </ul>
+    </div>
+  </section>
+  <section>
+    <div class="cards-container">
+      <ul class="cards-list">
+        <li class="cards-item">
+          <img src="images/SnapshootPortfolio.jpg" alt="Snapshoot">
+          <div class="title-container">
+            <h2>Tonic</h2>
+            <div class="list">
+
+              <span class="list-item-item1">Canopy</span>
+              <span class="list-item">back end Dev</span>
+              <span class="list-item">2015</span>
+            </div>
+          </div>
+          <div class="text-container">
+            <p>A daily selection of privately
+              personalized reads; no accounts or
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
```

Comment:
Each card should have white background:

Yours:
<img width="364" alt="image" src="https://user-images.githubusercontent.com/87897753/159586393-d651ad91-a566-4415-a2d9-069ff3bd9ade.png">

Figma's:
<img width="230" alt="image" src="https://user-images.githubusercontent.com/87897753/159586410-c3efb04f-93bc-478a-b81b-0845f2742072.png">